bugheist/settings.py

```diff
@@ -263,7 +263,7 @@
 
 # Simplified static file serving.
 # https://warehouse.python.org/project/whitenoise/
-STATICFILES_STORAGE = 'whitenoise.django.GzipManifestStaticFilesStorage'
+#STATICFILES_STORAGE = 'whitenoise.django.GzipManifestStaticFilesStorage'
```

Comment:
@rtgdk - Is there any other way to fix it while keeping the whitenoise functionality ? Because it closes the entire whitenoise mechanism while solving issue for one image.

Reply:
The fix is either to remove the line in the CSS which references the missing file, or to copy the file into the right place so it's no longer missing.
I'll look into it.

Reply:
Or we can comment the `django.contrib.staticfiles` . We can also stop heroku from running collect static while pushing by  `heroku config:set DISABLE_COLLECTSTATIC=1`